<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyScenarioDlg</class>
 <widget class="QWidget" name="ModifyScenarioDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scenario</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lbName">
         <property name="text">
          <string comment="Name of the scenario">Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KLineEdit" name="leName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Scenario version:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KIntSpinBox" name="sbScenarioVersion">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbMinVersion">
         <property name="text">
          <string>Min Simon version:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KLineEdit" name="leMinVersion">
         <property name="inputMask">
          <string>9.9xxxxxxx0; </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="KIconButton" name="pbIcon">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>61</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="lbMaxVersion">
       <property name="text">
        <string>Max Simon version:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="KLineEdit" name="leMaxVersion">
         <property name="toolTip">
          <string>You can leave this field empty to set no maximum version.</string>
         </property>
         <property name="inputMask">
          <string>0x0xxxxxxx0; </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>65</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbLicense">
       <property name="text">
        <string>License:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KComboBox" name="cbLicense">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>10</number>
       </property>
       <item>
        <property name="text">
         <string comment="Artistic license">Artistic 2.0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BSD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CPL 1.0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Creative Commons by</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Creative Commons by-nc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Creative Commons by-nc-nd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Creative Commons by-nc-sa</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Creative Commons by-nd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Creative Commons by-sa</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GFDL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GPL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LGPL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MIT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Proprietary License</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QPL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="X11 license">X11</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbAuthors">
       <property name="text">
        <string>Authors:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QListWidget" name="lwAuthors"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="pbAddAuthor">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="isDragEnabled" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="pbRemoveAuthor">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIconButton</class>
   <extends>QPushButton</extends>
   <header>kicondialog.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
